pom.xml

```diff
@@ -112,7 +112,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jitsi-media-transform</artifactId>
-      <version>1.0-135-g687d806</version>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
This doesn't need a JMT update, right?

Reply:
No, you're right - I was task switching between too many different PRs when I wrote this.